queue_job/readme/CONFIGURE.rst

```diff
@@ -41,3 +41,14 @@
 
 .. [1] It works with the threaded Odoo server too, although this way
        of running Odoo is obviously not for production purposes.
+
+* Be sure to check out *Jobs Garbage Collector* CRON and change *enqueued_delta* and *started_delta* parameters to your needs.
+
+  * enqueued_delta: Spent time in minutes after which an enqueued job is considered stuck.
+  * started_delta: Spent time in minutes after which a started job is considered stuck.
+                   This parameter should not be less than *--limit-time-real / 60.0* parameter in your configuration.
```

Comment:
issue: this is not being rendered properly because of the indentation, and there are missing docs. Consider this suggestion please:

```suggestion
  * ``enqueued_delta``: Spent time in minutes after which an enqueued job is considered stuck.
    Set it to 0 to disable this check.
  * ``started_delta``: Spent time in minutes after which a started job is considered stuck.
    This parameter should not be less than ``--limit-time-real / 60.0`` parameter in your configuration.
    Set it to 0 to disable this check. Set it to -1 to automate it, based in the server's ``--limit-time-real`` config parameter.
```